src/list/views/ListView.js

```diff
@@ -154,7 +154,6 @@ let ListView = Marionette.LayoutView.extend({
         });
 
         this.visibleCollectionRegion.show(this.visibleCollectionView);
-        this.__handleResizeInternal();

```

Comment:
This line is duplicate of 144 (they were both created under the same commit) and causes grid sizing bug under certain circumstances. What was the goal of this line?